lib/pages/chat_details/chat_details_edit.dart

```diff
@@ -443,6 +450,24 @@ class ChatDetailsEditController extends State<ChatDetailsEdit>
     avatarAssetEntity = null;
   }
 
+  String? getErrorMessage(String content) {
+    return content == room?.name
+        ? null
+        : verifyNameInteractor.execute(
+            content,
+            [EmptyNameValidator(), NameWithSpaceOnlyValidator()],
+          ).fold(
+            (failure) {
+              if (failure is VerifyNameFailure) {
+                return failure.getMessage(context);
+              } else {
+                return null;
+              }
+            },
+            (success) => null,
+          );
+  }
```

Comment:
```suggestion
  String? getErrorMessage(String content) {
    if (content == room?.name) {
      return null;
    }

    final result = verifyNameInteractor.execute(
      content,
      [EmptyNameValidator(), NameWithSpaceOnlyValidator()],
    );

    return result.fold(
      (failure) =>
          failure is VerifyNameFailure ? failure.getMessage(context) : null,
      (success) => null,
    );
  }
```

Reply:
Hi @KhaledNjim why resolved this conversation?

Reply:
its fixed

Reply:
What's different? What I mean is don't use ternary operators like that for many conditions. PLSSSSSS